cognite/neat/app/api/routers/rules.py

```diff
@@ -84,3 +91,47 @@ def get_rules(
         "error_text": error_text,
         "src": src,
     }
+
+
+@router.get("/api/rules/from_file")
+def get_original_rules_from_file(
+    file_name: str | None = None,
+):
+    """Endpoing for retrieving raw transformation from file"""
+    path = Path(neat_app.config.rules_store_path, file_name)
+    rules = parse_rules_from_excel_file(path)
+    return Response(content=rules.model_dump_json(), media_type="application/json")
+
+
+@router.get("/api/rules/from_workflow")
+def get_original_rules_from_workflow(
+    workflow_name: str,
+):
+    """Endpoing for retrieving transformation from memmory"""
+    workflow = neat_app.workflow_manager.get_workflow(workflow_name)
+    context = workflow.get_context()
+    rules = context["RulesData"].rules
+    return Response(content=rules.model_dump_json(), media_type="application/json")
+
+
+@router.post("/api/rules/model_and_transformations")
+def post_original_rules(request: dict):
+    """Endpoing for updating transformation rules via API"""
+    request["metadata"]["namespace"] = Namespace(request["metadata"]["namespace"])
+    metadata = Metadata(**request["metadata"])
+    classes: dict[str:Class] = {}
+    for class_, val in request["classes"].items():
+        classes[class_] = Class(**val)
+    properties: dict[str:Property] = {}
+
+    for prop, val in request["properties"].items():
+        val["resource_type_property"] = []
+        properties[prop] = Property(**val)
+
+    prefixes: dict[str, Namespace] = {}
+    for prefix, val in request["prefixes"].items():
+        prefixes[prefix] = Namespace(val)
+
+    TransformationRules(metadata=metadata, classes=classes, properties=properties, prefixes=prefixes, instances=[])
```

Comment:
How is this updating transformation rules?`TransformationRules` class is instantiated here but not stored anywhere?